Find a functionally equivalent code:android.graphics.drawable.AnimationDrawable.isRunning:COMMENT
Method Modifier: public      
Comment:/**
 * <p>Indicates whether the animation is currently running or not.</p>
 *
 * @return true if the animation is running, false otherwise
 */

@@ -1,3 +1,3 @@
 {
-    return mCurFrame > -1;
+    return mRunning;
 }

